pandas/core/frame.py

```diff
@@ -8972,7 +8972,7 @@ def mode(self, axis=0, numeric_only=False, dropna=True) -> DataFrame:
 
         By default, missing values are not considered, and the mode of wings
         are both 0 and 2. The second row of species and legs contains ``NaN``,
-        because they have only one mode, but the DataFrame has two rows.
+        because they have only one mode, but the resulting DataFrame has two rows.
```

Comment:
How about
```
Because the resulting DataFrame has two rows, the second row of ``species`` and ``legs`` contains ``NaN``
```
? (you may need to wrap this around multiple lines)

Reply:
That is more descriptive. Totally agreed

Reply:
Cool - could you also put a more descriptive title for your pull request please?